test/unit/modules/transactions.js

```diff
@@ -13,15 +13,12 @@
  */
 'use strict';
 
-var chai = require('chai');
-var expect = require('chai').expect;
 var async = require('async');
-var sinon = require('sinon');
+var rewire = require('rewire');
```

Comment:
👌

Reply:
I think this is because i removed the rewire below and clubbed all required together, so thats the reason.

Reply:
Yes, nicer to have all required `node_modules` together.